### ISSUE TITLE
ReturnedData.as_dataframe(expand=True) failing with AUX_OBS data

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/VirES-Python-Client#54]
After downloading AUX_OBS data, the invocation of ReturnedData.as_dataframe() method fails if I set `expand=True`. E.g. the following code:

```python
import numpy as np
from viresclient import SwarmRequest

START_DATE = '2016-01-01'
END_DATE = '2016-01-02'

request = SwarmRequest()
request.set_collection('SW_OPER_AUX_OBSM2_')
request.set_products(measurements=request.available_measurements('SW_OPER_AUX_OBSM2_'))
df = request.get_between(START_DATE, END_DATE).as_dataframe(expand=True)
```

raises an `IndexError` exception:

```python
[1/1] Processing:  100%|██████████|  [ Elapsed: 00:01, Remaining: 00:00 ]
      Downloading: 100%|██████████|  [ Elapsed: 00:00, Remaining: 00:00 ] (10.24MB)
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-10-f95a9b846568> in <module>
      8 request.set_collection('SW_OPER_AUX_OBSM2_')
      9 request.set_products(measurements=request.available_measurements('SW_OPER_AUX_OBSM2_'))
---> 10 df = request.get_between(START_DATE, END_DATE).as_dataframe(expand=True)

/opt/conda/lib/python3.7/site-packages/viresclient/_data_handling.py in as_dataframe(self, expand)
    565         """
    566         return pandas.concat(
--> 567             [d.as_dataframe(expand=expand) for d in self.contents])
    568 
    569     def as_xarray(self):

/opt/conda/lib/python3.7/site-packages/viresclient/_data_handling.py in <listcomp>(.0)
    565         """
    566         return pandas.concat(
--> 567             [d.as_dataframe(expand=expand) for d in self.contents])
    568 
    569     def as_xarray(self):

/opt/conda/lib/python3.7/site-packages/viresclient/_data_handling.py in as_dataframe(self, expand)
    419         elif self.filetype == 'cdf':
    420             with FileReader(self._file) as f:
--> 421                 df = f.as_pandas_dataframe(expand=expand)
    422         return df
    423 

/opt/conda/lib/python3.7/site-packages/viresclient/_data_handling.py in as_pandas_dataframe(self, expand)
    190             if len(vector_data.shape) > 2:
    191                 raise NotImplementedError("{}".format(column))
--> 192             if vector_data.shape[1] != len(suffixes):
    193                 raise NotImplementedError("{}".format(column))
    194             for i, suffix in enumerate(suffixes):

IndexError: tuple index out of range
```

**Comment by smithara** [comment]
Thanks for the report!
It has something to do with the `"Quality"` variable.

```python
from viresclient import SwarmRequest
request = SwarmRequest('https://staging.viresdisc.vires.services/ows')
request.set_collection("SW_OPER_AUX_OBSM2_:ABK")
request.set_products(["B_NEC", "Quality"])
data = request.get_between("2016-01-01", "2016-01-02")
data.as_dataframe(expand=True)
```
only fails when `"Quality"` is introduced.

I'm looking into it

**Comment by smithara** [comment]
The problem was due to a conflict with the `"Quality"` variable in the `"AOB_FAC"` collections (which is 2-dimensional, and therefore it was trying to expand it).

I've made a bit of an ugly fix in https://github.com/ESA-VirES/VirES-Python-Client/commit/cdf851ed91cfda095d8c621f91532163b71e463e and will be released in 0.7.1 but really need to reorganise the way the way the configuration of the data variables is done in general.
